src/core/chmstate_mod.F90

```diff
@@ -1,3 +1,11 @@
+!> \defgroup ChmState_Mod ChmState Module
+!! \brief Contains the ChmStateType data type and related subroutines and functions.
+!!
+!! The ChmState_Mod module contains the ChmStateType data type and related subroutines and functions for managing the state of the chemical model.
+!!
+!! \author Barry baker
+!! \date 05/2023
```

Comment:
With Git, maybe not necessary?

Reply:
not sure if this will appear in the docstrings later on?

Reply:
Maybe, but if we need the info we can always get it from Git.

Reply:
removed